laravel/app/Http/Controllers/ScheduleDataController.php

```diff
@@ -298,7 +298,8 @@ public function getChoice() {
     }
 
     public function parseCaseProcedure($case){
-        $case = preg_replace('/[0-9:()\[\]]/', '', $case);
+        $case = str_replace(' [', '', $case);
+	    $case = preg_replace('/[0-9:()\[\]]/', '', $case);
         return $case;
```

Comment:
I think a comment could go quite a long way here, considering that many people aren't particularly good at reading REGEX.